user/gdu/template.py

```diff
@@ -1,16 +1,23 @@
 pkgname = "gdu"
-pkgver = "5.30.1"
-pkgrel = 6
+pkgver = "5.31.0"
+pkgrel = 0
 build_style = "go"
-make_build_args = ["./cmd/gdu"]
+make_build_args = [
+    f"-ldflags=-X github.com/dundee/gdu/v5/build.Version={pkgver}",
+    "./cmd/gdu",
+]
 # expects writing to /xyzxyz to give eperm instead of erofs (which happens in --ro bwrap)
```

Comment:
this comment is only about `TestOutputFileError` from what it seems, please update it to clarify that

Reply:
sure